melodic/distribution.yaml

```diff
@@ -6600,6 +6610,16 @@ repositories:
       url: https://gitlab.com/jlack/rdl.git
       version: master
     status: developed
+  realsense-ros:
```

Comment:
In kinetic, this key is called "realsense2_camera" instead of "realsense-ros".  Any reason for the discrepency?  Even if we do want to rename the key, we should use an underscore instead of a dash here.